site2/docs/admin-api-clusters.md

```diff
@@ -4,14 +4,24 @@ title: Managing Clusters
 sidebar_label: Clusters
 ---
 
+> **Important**
+>
+> This page only shows **some frequently used operations**.
+>
+> - For the latest and complete information about `pulsar-admin`, including commands, flags, descriptions, and more, see [here](https://pulsar.apache.org/tools/pulsar-admin/).
```

Comment:
Using a link with only the text "here" or "click here" is generally not good for UX. An alternative would be something like:

> For the latest and most complete information about `pulsar-admin`, see [the pulsar-admin documentation](https://pulsar.apache.org/tools/pulsar-admin/)

Reply:
I'm actually not sure what we want to call this page, but it would be useful to have a clear name. Should we just call it the "documentation"? or maybe it's a "manual"? I kind of like the term manual since it's essentially a man page.

Reply:
Hi @gmethvin, thanks for your suggestion. I've updated them to "xxx, see [pulsar-admin doc](xxx) /  see [REST API doc](xxx) / see [Java admin API doc](xxx).

Reply:
Hi @gmethvin I've incorporated your comments, PTAL.

Since it's ok as-is, if no other comments, could you please help approve? Then users can get the info quickly.
We can do some research on the duplicated text and optimize the user experience later.